### PR TITLE
remove redundant settings from `CollectFrameDataFromAssetEntity`

### LINKED ISSUE #50: Remove redundant settings or implement missing logic
### Is there an existing issue for this?

- [x] I have searched the existing issues and added correct labels.

### Please describe the feature you have in mind and explain what the current shortcomings are?

some plugins like `CollectFrameDataFromAssetEntity` don't implement using optional settings but they have optional settings in the addon settings.

### Suggested implementation?

A. Remove redundant settings
B. Implement missing logic

### Describe alternatives you've considered:

_No response_

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
resolve #50 
This PR removes `CollectFrameDataFromAssetEntity` settings completely.  this plugin was previously Optional, but that got later removed https://github.com/ynput/OpenPype/commit/e90d227a234fec278b214c3ed4086350403eda80

## Testing notes:
1. everything should work as before. 
2. Please report if the missing logic should be implemented instead of removing the redundant settings.
3. Please report if the PR is not backward compatible (copy settings should work fine without implementing conversions)

### CODE REVIEW

File: server/settings/publish_plugins.py [hunk 1]
```diff
@@ -11,6 +11,11 @@ class ValidatePluginModel(BaseSettingsModel):
     active: bool = SettingsField(True, title="Active")
 
 
+class MiniValidatePluginModel(BaseSettingsModel):
+
```
Comment (BigRoy) [inline server/settings/publish_plugins.py:15]:
```suggestion
```

I also think a better name is `EnabledStateModel` instead of `MiniValidatePluginModel`

Comment (MustafaJafar) [reply server/settings/publish_plugins.py]: Thank you
https://github.com/ynput/ayon-traypublisher/pull/51/commits/fc5dcbb47bea50492f3b0ad341a07990fc9c3be3

**Comment by iLLiCiTiT** [comment]
Shouldn't we fix it so the plugin shows the option in UI, instead of removing the settings?

**Comment by MustafaJafar** [comment]
> Shouldn't we fix it so the plugin shows the option in UI, instead of removing the settings?

@jakubjezek001 @kalisp @robin-ynput any idea about the intended behavior of this plugin `CollectFrameDataFromAssetEntit`?

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**